Wrapping/Generators/Python/itk-stubs/BaseClasses.pyi

```diff
@@ -0,0 +1,407 @@
+from itkConfig import ITK_GLOBAL_VERSION_STRING as __version__
+from itk.support.extras import *
+from itk.support.init_helpers import *
+from itk.support.types import * # itkCType, F, D, UC, US, UI, UL, SL, LD, ULL, SC, SS, SI, SLL, B
```

Comment:
We want to use explicit imports, not `import *` as a best practice. There may be types  that conflict with the classes defined here.

Reply:
The convention we used for `itk.support.types` is `import itk.support.types as itkt`.

Reply:
Thanks for the feedback! In this update I returned it to the way that it appears in `__init__.py` so that the suggestions would match what is available.